workflow/rules/phenotype_process.smk

```diff
@@ -13,13 +13,29 @@ rule apply_ic_field_phenotype:
     script:
         "../scripts/phenotype_process/apply_ic_field_phenotype.py"
 
+# Align phenotype images
+rule align_phenotype:
+    conda:
+        "../envs/phenotype_process.yml"
+    input:
+        PHENOTYPE_PROCESS_OUTPUTS["apply_ic_field_phenotype"]
+    output:
+        PHENOTYPE_PROCESS_OUTPUTS_MAPPED["align_phenotype"]
+    params:
+        align=config["phenotype_process"]["align"],
+        target=lambda wildcards: config["phenotype_process"]["target"] if config["phenotype_process"]["align"] else None,
+        source=lambda wildcards: config["phenotype_process"]["source"] if config["phenotype_process"]["align"] else None,
+        riders=lambda wildcards: config["phenotype_process"]["riders"] if config["phenotype_process"]["align"] else None,
+        remove_channel=lambda wildcards: config["phenotype_process"]["remove_channel"] if config["phenotype_process"]["align"] else None,
```

Comment:
This is sick, good idea to dynamically include params here. But, do we need `lambda wildcards:` here? It seems like we shouldn't need it if we aren't referencing the `wildcards` var.